### ISSUE TITLE
LG webOS TV browser: top bar doesn't hide completely

### ISSUE THREAD

**Issue by n1trux** [jellyfin/jellyfin-web#2756]
since webOS 3.23.05 the top bar of the LG browser doesn't hide the bar completely, it just slides away and there's still a 20px or so wide stripe on the top of the screen (see attached screenshot).

Maybe if Jellyfin went into fullscreen mode when playing media this issue could go away. There's no way to go into full screen mode within the browser itself, as far as I can see.

**Steps To Reproduce**
1. open LG browser
2. go to Jellyfin
3. start playing any media
4. bar slides up but is still visible

**Expected Behavior**
The bar should disappear because Jellyfin goes into full screen mode when it detects a TV browser.

**Screenshots**
Bar shown:
![bar_shown](https://user-images.githubusercontent.com/7255542/123626160-ccae8d80-d810-11eb-9b84-714c6bcd281b.jpg)
Bar "hidden" (more like: slid up):
![bar_hidden](https://user-images.githubusercontent.com/7255542/123626157-cc15f700-d810-11eb-96d6-924cc66edf94.jpg)

**System (please complete the following information):**
 - Platform: LG 55NANO913NA TV
 - Browser: LG browser (version not visible anywhere, OS version 3.23.05)
 - Jellyfin Version: 10.7.6

**Comment by hundehausen** [comment]
It goes away after some time on my LG OLED.

**Comment by FluffyDiscord** [comment]
Once it goes away, in about 5 mins, subtitles go away too.

**Comment by hundehausen** [comment]
And at some point the screen saver kicks in.

**Comment by imchasingshadows** [comment]
The new WebOS browser appears to have dropped the `Web0S` part from the useragent causing jellyfin to not correctly detect the new browser. In addition to this jellyfin hides the fullscreen toggle when the browser is detected to be a tv

the useragent im seeing in my access logs;
`Mozilla/5.0 (Linux; NetCast; U) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36 SmartTV/10.0 Colt/2.0`

I was able to resolve the issue locally by adding a new variable to the browser object
```js
browser.js:237

browser.web0s = userAgent.toLowerCase().indexOf('Web0S'.toLowerCase()) !== -1;
// im not sure which part of the useragent is unique but this seemed to work
browser.web0sChromium = userAgent.toLowerCase().indexOf('SmartTV'.toLowerCase()) !== -1;
```

And allowed the player to show the fullscreen toggle
```js
apphosts.js:130

if (browser.web0sChromium) {
  return true;
}
if (browser.tv) {
        return false;
}
```


This allowed me to enable the fullscreen-toggle which puts the video into "true" fullscreen resolving the small grey bar and subtitle issues. I'd prefer if the player would go into fullscreen automatically but im not sure if thats possible.

I'm not sure if this is the optimal way to resolve this issue but i think its a good starting point for someone with knowledge of the codebase.

Let me know if i can do anything more to help

**Comment by n1trux** [comment]
Thanks. Reporting Bugs to LG is very very annoying to me. The support site didn't work properly for me and I gave up. Can someone tell me an easy way to report bugs or do they really just not want to service their users?

**Comment by TheLastProject** [comment]
I updated my Docker to 10.8.0 Alpha 5 and this issue appears to be fixed on my LG WebOS TV (running WebOS 03.21.20).
